### PR TITLE
solving deprecated python code ahead of cloud deployment

### PR DESCRIPTION
**Pull request by iglesu**

### CODE REVIEW

File: 03_ScienceExamples/EmLineGalaxies/01_EmLineGalaxies_SpectraStack.ipynb
Comment (jacquesalice) [inline 03_ScienceExamples/EmLineGalaxies/01_EmLineGalaxies_SpectraStack.ipynb:1]: Notebook ran as expected with no errors or warnings. My only comment is to change the version date in the first cell to 20230607 (or whatever date you changed the notebook).

File: 03_ScienceExamples/GOGREEN_GalaxiesInRichEnvironments/3_GOGREENDr1ClusterMembership.ipynb
Comment (jacquesalice) [inline 03_ScienceExamples/GOGREEN_GalaxiesInRichEnvironments/3_GOGREENDr1ClusterMembership.ipynb:1]: In the plot under the section titled "Select cluster with the highest dynamical mass", the black star that is supposed to indicate the position of the selected cluster isn't showing up on the plot...

Screen Shot 2023-06-07 at 4 01 20 PM


It's _supposed_ to look like this:

Screen Shot 2023-06-07 at 4 02 15 PM

Also change the version date in the first cell to 20230607 (or whatever date you changed the notebook).

File: 03_ScienceExamples/GOGREEN_GalaxiesInRichEnvironments/4_GOGREENDr1Spec1DRedshift.ipynb
Comment (jacquesalice) [inline 03_ScienceExamples/GOGREEN_GalaxiesInRichEnvironments/4_GOGREENDr1Spec1DRedshift.ipynb:1]: Notebook ran as expected with no errors or warnings. My only comment is to change the version date in the first cell to 20230607 (or whatever date you changed the notebook).

**Review by jacquesalice — CHANGES_REQUESTED** [review]
I left a comment on each of the notebooks

**Comment by rnikutta** [comment]
Hi @iglesu and @jacquesalice I've pushed a commit to this branch, fixing all issues (I think):
- Deprecation of `np.filter.filterwarnings()`: instead of a global `import warnings; warnings.filterwarnings('ignore')` now using the more narrow `np.seterr(divide='ignore')` or `np.seterr(invalid='ignore')` or both.
- Deprecation of `pd.append()`. Now using `pd.concat()`.
- Fixed a bug in a function in NB 3_GOGREENS..., and its call, which prevented a star symbol to be plotted.
- Replaced the `label=str(np.int(snr))` issue with the cleaner `label="%d" % snr`
- Updated version stamps in these four NBs.
- Added HTML rendered versions too.

Tested all four NBs on gp12 and GCP, they now run fine. Will merge this PR.

**Pull request merged**